app/src/common/shared/org/mozilla/vrbrowser/SettingsStore.java

```diff
@@ -231,4 +232,15 @@ public void setMaxWindowHeight(int aMaxWindowHeight) {
         editor.commit();
     }
 
+    public String getEnvironment() {
+        return mPrefs.getString(
```

Comment:
Are pref functions thread safe or do we need to synchronize getEnvironment and setEnvironment since they are called in both UI thread and GL thread?

Reply:
Okay, I see they are.